### DISCUSSION TITLE
Sent me to the wrong license purchase panel need a chrome one not fire fox.

### DISCUSSION THREAD

**Discussion by Nuclarwharhead** [aclap-dev/video-downloadhelper#1378] [License & Purchase requests]
### What's your issue?

So, I'm using Opera Gx as my browser; I decided to upgrade finally to premium, it sent me to the Firefox license purchase page and I thought nothing of it, maybe opera was weird and uses some Firefox stuff, no biggie till I go to use the license to upgrade my stuff and it does work, it wants a chrome license but it only sends me to the firefox page. I want either a full refund of my 28.5$ purchase or a chrome license key.

### The 10 first digits (only 10) of your license number or email address

44771dab40

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by Nuclarwharhead** [comment]
and yes I did contact support no luck in getting them to respond so far.

**Comment by jcfrog** [comment]
Sorry we can't swap licenses.
What I can do is refund your Firefox premium so that you can purchase Chrome premium if you want to.
Do you want me to do this ?

**Reply by Nuclarwharhead** [reply]
That would be greatly appreciated if Not already Put in progres

**Reply by jcfrog** [reply]
You have been refunded. Should be available in a few business days.

**Reply by DigitalSolaire** [reply]
Hi, the same thing also happened for me as well. Thought I was buying a chrome license but didn't know it was a firefox license. I would also like a refund if possible to buy the chrome license.

**Reply by jcfrog** [reply]
@DigitalSolaire please send us an email at licenses@downloadhelper.net
